style.css

```diff
@@ -751,34 +600,938 @@ input[type="email"]::placeholder {
+    padding-left: 0%;
+    width: 295px;
+    height: 60px;
+    display: flex;
+    align-items: center;
+  }
+
+  li.tag-button {
+    display: inline-block;
+    background-color: #ebebff;
+    padding: 4px 12px;
+    color: var(--color-main);
+    border-radius: 8px;
+    font-size: 12px;
+    margin-right: 5px;
+  }
+
+  div.pro-button {
+    width: 295px;
+    height: 72px;
+    display: flex;
+    align-items: center;
+  }
+
+  a.button {
+    text-decoration: none;
+    color: var(--color-main);
+    border: 1px solid var(--color-main);
+    border-radius: 8px;
+    font-size: 17px;
+    width: 124px;
+    height: 48px;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+  }
+
+  a.button:hover {
+    text-decoration: none;
+    background-color: var(--color-main);
+    color: white;
+    border-radius: 8px;
+    font-size: 17px;
+    width: 124px;
+    height: 48px;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+  }
+
+  a.button:active {
+    text-decoration: none;
+    background-color: var(--button-pressed);
+    color: white;
+    width: 124px;
+    height: 48px;
+    font-size: 17px;
+    border-radius: 8px;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+  }
+
+  section.project-b {
+    grid-column-start: 2;
+    grid-column-end: 3;
+    background-color: white;
+    height: 596px;
+    width: 327px;
+    border-radius: 16px;
+    border: 1px solid var(--cards-border);
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    margin: 0% auto;
+  }
+
+  .pro2a {
+    height: 220px;
+    width: 295px;
+    margin: 12px 0;
+    background-image: url(./images/projects2.png);
+    background-size: 295px 220px;
+    border-radius: 8px;
+    margin-top: 16px;
+  }
+
+  div.pro-2b {
+    height: 124px;
+    width: 295px;
+    display: flex;
+    align-items: center;
+    flex-direction: column;
+  }
+
+  div.project-title-b {
+    color: var(--text-color-titles);
+    font-size: 32px;
+    line-height: 44px;
+    width: 295px;
+    height: 84px;
+  }
+
+  .pro3a {
+    height: 220px;
+    width: 295px;
+    margin: 12px 0;
+    background-image: url(./images/projects3.png);
+    background-size: 295px 220px;
+    border-radius: 8px;
+    margin-top: 16px;
+  }
+
+  .pro4a {
+    height: 220px;
+    width: 295px;
+    margin: 12px 0;
+    background-image: url(./images/projects4.png);
+    background-size: 295px 220px;
+    background-repeat: no-repeat;
+    border-radius: 8px;
+    margin-top: 16px;
+  }
+
+  #about {
+    height: 1125.15px;
+    background-color: white;
+    border-top-right-radius: 100px;
+    display: flex;
+    flex-direction: column;
+    justify-content: flex-start;
+    align-items: center;
+    padding: 20px 20px;
+    margin: 0% auto;
+    width: 375px;
+    border: 4px solid white;
+    padding-top: 35%;
+  }
+
+  #about .about-a {
+    width: 327px;
+    height: 409px;
+    margin-bottom: 24px;
+  }
+
```

Comment:
- [ ] Kindly make this left side have 40% width and the right side have 60% width on Desktop screen

Your Design
![image](https://user-images.githubusercontent.com/50941074/160214935-b1020f5a-e4a2-4085-aa13-50d0f1cdf044.png)

Figma Design
![image](https://user-images.githubusercontent.com/50941074/160215024-4fe0f3f2-620a-472c-88dd-95be920378b7.png)